src/Params/EnvelopeParams.cpp

```diff
@@ -324,17 +324,17 @@ void EnvelopeParams::init(zyn::consumer_location_t _loc)
 {
     switch(loc = _loc)
     {
-        case ad_global_amp:    ADSRinit_dB(0, 0.01, 127, 0.025); break;
-        case ad_global_freq:   ASRinit(64, 0.025, 64, 0.5); break;
+        case ad_global_amp:    ADSRinit_dB(0.00f, 0.127f, 127, 0.041f); break;
+        case ad_global_freq:   ASRinit(64, 0.254f, 64, 0.499f); break;
         case ad_global_filter:
-        case sub_filter:       ADSRinit_filter(64, 0.01, 64, 0.025, 0.05, 64); break;
-        case ad_voice_amp:     ADSRinit_dB(0, 7.0, 127, 7.0); break;
-        case ad_voice_freq:    ASRinit(30, 0.01, 64, 0.5); break;
-        case ad_voice_filter:  ADSRinit_filter(90, 0.025, 40, 0.025, 0.009, 40); break;
-        case ad_voice_fm_freq: ASRinit(20, 3.62, 40, 1.876); break;
-        case ad_voice_fm_amp:  ADSRinit(80, 90, 127, 100); break;
-        case sub_freq:         ASRinit(30, 0.025, 64, 0.5); break;
-        case sub_bandwidth:    ASRinit_bw(100, 0.97, 64, 0.5); break;
+        case sub_filter:       ADSRinit_filter(64, 0.127f, 64, 0.970f, 0.499f, 64); break;
+        case ad_voice_amp:     ADSRinit_dB(0, 6.978f, 127, 6.978f); break;
```

Comment:
Just to make clear for the reader that it's a float:
```suggestion
        case ad_voice_amp:     ADSRinit_dB(0.f, 6.978f, 127, 6.978f); break;
```